src/main/java/com/cen4010/bookstore/book/Book.java

```diff
@@ -7,43 +7,115 @@
 import java.util.UUID;
 import javax.persistence.Column;
 import javax.persistence.Entity;
-import javax.persistence.GeneratedValue;
-import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.ManyToMany;
-import javax.persistence.SequenceGenerator;
 import org.hibernate.annotations.Type;
 
+
 @Entity
 public class Book {
 
   @Id
   @Column(name = "id")
   @Type(type = "uuid-char")
-  private UUID isbn;
+  private UUID id;
+  private String ISBN;
   private String name;
-
+  private String description;
+  private double price;
+  private String author;
+  private String genre;
+  private String publisher;
+  private int yearPublished;
+  private int copiesSold;
+  //private final double rating;
```

Comment:
Let's remove commented code from this. I think ratings will need to be derived- i.e. `SELECT *id FROM reviews GROUP BY bookID ORDER BY avg(ratingNumber) DESC LIMIT 5;`. Or just do that in Java since tbh I'm not sure how to do this in JPA